float64[] pose_joint_1
float64[] pose_joint_2
float64[] pose_joint_3
float64[] pose_joint_4
float64[] pose_joint_5
float64[] pose_joint_6

float64[] speed_joint_1
float64[] speed_joint_2
float64[] speed_joint_3
float64[] speed_joint_4
float64[] speed_joint_5
float64[] speed_joint_6

float64[] acc_joint_1
float64[] acc_joint_2
float64[] acc_joint_3
float64[] acc_joint_4
float64[] acc_joint_5
float64[] acc_joint_6


